addon/components/nrg-sidebar-menu/component.js

```diff
@@ -24,9 +24,6 @@ export default Component.extend({
   hasVisibleFooterItems: notEmpty('visibleFooterItems'),
 
   actions: {
-    clickedLink() {
```

Comment:
I would keep it the way it was for this component. 

This is because the if someone didn't implement the `clicked` function, there would be a template error.

Reply:
```js
clickedLink((item) {
  if (this.clicked) {
    this.clicked(item);
  }
}
```